src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -165,7 +196,44 @@ public void run() {
         importDirectory(importContainerDirectory);
 
         importMembershipResources();
-        importLogger.info("Finished import... {} resources imported", successCount.get());
+
+        // loop through for nested member/membership resources
+        if (memberResources.size() > 0) {
+            final List<URI> referenceResources = memberResources.stream().collect(toList());
+            memberResources.clear();
+            importMemberResources(referenceResources);
+        }
+    }
+
+    /**
+     * Discovery the base url for the resource and the source being importing.
+     */
+    private void discoveryImportBase() {
+        if (repositoryRoot == null) {
+            String sourceRoot = config.getSource().toString();
+            String resourceRoot = config.getResource().toString();
+            repositoryRoot = importContainerDirectory.getParentFile();
+            if (memberResources.size() > 0) {
+                // A container that includes more than two rdf files with member resource
+                while (listRdfFiles(repositoryRoot).length > 1) {
+                    sourceRoot = sourceRoot.substring(0, sourceRoot.lastIndexOf("/"));
+                    resourceRoot = resourceRoot.substring(0, resourceRoot.lastIndexOf("/"));
+                    repositoryRoot = repositoryRoot.getParentFile();
+                }
+            }
+
```

Comment:
Please add `debug` logging that shows what the values of `sourceBase` and `resourceBase` are.